generators/app/validator.js

```diff
@@ -1,4 +1,5 @@
 const nameRegex = /^[a-z0-9][a-z0-9\-]*$/i;
+const publisherRegex = /^(@[a-z0-9-~][a-z0-9-._~]*\/)[a-z0-9-~][a-z0-9-._~]*$/;
```

Comment:
Is this regex somewhere in the lens repo?

Reply:
no, it's in this repo only

Reply:
Okay. I think we should be forcing this in Lens too. But that is for the future I guess.